openhcl/hcl/src/ioctl.rs

```diff
@@ -1561,6 +1561,35 @@ impl MshvHvcall {
 
         status.result()
     }
+
+    /// Request a VBS VM report from the host VSM.
+    pub fn vbs_vm_call_report(
+        &self,
+        report_data: &[u8],
+    ) -> Result<[u8; hvdef::hypercall::VBS_VM_MAX_REPORT_SIZE], HvError> {
+        assert!(report_data.len() <= hvdef::hypercall::VBS_VM_REPORT_DATA_SIZE);
```

Comment:
what is this field here, why is it an assert? who's supplying this `report_data`?

Reply:
changed to if condition --- the intention is for input validation against the limit defined by hypercall